spec/index.bs

```diff
@@ -9534,6 +9534,9 @@ dictionary GPURenderPassDepthStencilAttachment {
         to prior to executing the render pass. Ignored if {{GPURenderPassDepthStencilAttachment/stencilLoadOp}}
         is not {{GPULoadOp/"clear"}}.
 
+        Note: The value will always be converted to the type of the stencil aspect of |view| by taking the same
```

Comment:
"Note:" [makes it non-normative](https://gpuweb.github.io/gpuweb/#w3c-conventions), but this should be normative. Eventually, there will be algorithm text describing exactly how the clear occurs, but in the mean time this serves as our normative definition of that.

```suggestion
        The value will always be converted to the type of the stencil aspect of |view| by taking the same
```